test/jit/test_functional_blocks.py

```diff
@@ -127,6 +128,59 @@ def test_intermediary_use():
         FileCheck().check_count("aten::add_", 1).run(graph)
         self.assertEqual(test_intermediary_use(), fn())
 
+    def test_remove_mutation_if_output(self):
```

Comment:
aside: I wish these tests were written in cpp. Here we are invoking the frontend, but our actual assertion checks the IR. It makes it hard to read the test and couples the test to the frontend. I find myself wanting to see the "pre-functionalization" IR directly any time I read one of these tests.